### ISSUE TITLE
Hydropic degeneration of chorionic villi

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10118]
**Preferred term label:**
Hydropic degeneration of chorionic villi

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C5394873

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Molar pregnancy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think this is at least a near synonym?
[Hydropic placenta HP:0011414](https://hpo.jax.org/app/browse/term/HP:0011414)

**Comment by MickeySegal** [comment]
[Hydropic placenta HP:0011414](https://hpo.jax.org/app/browse/term/HP:0011414) is a bundled term, but we can use it.

**Comment by pnrobinson** [comment]
OK, I will close this for now. We are going to be working on prenatal and placental phenotypes this year.
